index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="index.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/df740146e4.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    
+    
+    <nav class="nav">
+    <a href="#" class="logo">Portfolio</a>
+    <a href="#" class="menu"><i class="fa-solid fa-bars"></i></a>
+    </nav>
+    
+    <main class="main">
+        <h1 class="primaryHeading">
+            I'm Fahim Khan Glad to see you!
+        </h1>
+        <p class="mainContent">
+            I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+        </p>
+
+        <a href="#" class="connect">Let's Connect</a>
+        <ul class="icons">
+           <li><a href="#" class="iconList"><i class="fa-brands fa-twitter"></i></a></li>
+           <li><a href="#" class="iconList"><i class="fa-brands fa-linkedin-in"></i></a></li>
+           <li><a href="#" class="iconList"><i class="fa-brands fa-github"></i></a></li>
+           <li><a href="#" class="iconList"><i class="fa-brands fa-instagram"></i></a></li>
+        </ul>
+
```

Comment:
[OPTIONAL] Please note that your page does not match figma design margin and paddings for this section
Your page
<img width="1638" alt="Screenshot 2022-10-27 at 16 00 24" src="https://user-images.githubusercontent.com/48546992/198292124-81c696ac-cf3c-4724-abd0-cfd59c928add.png">

Figma design
<img width="376" alt="Screenshot 2022-10-27 at 16 00 36" src="https://user-images.githubusercontent.com/48546992/198292216-0d915e87-93a5-46e5-aebe-5056e6739867.png">